bin/lib/create.js

```diff
@@ -54,34 +54,12 @@ module.exports.createProject = (project_path, package_name, project_name, option
     fs.ensureDirSync(project_path);
 
     // copy templates/build-res directory ( recursive )
```

Comment:
Maybe, `// copy templates directory to the project path ( recursive )` would be more accurate?